package.json

```diff
@@ -36,37 +21,9 @@
     "koa-helmet": "1.0.0",
     "koa-mount": "1.3.0",
     "koa-route": "2.4.2",
-    "moment": "2.11.2",
```

Comment:
IMO having unused deps in package.json is a not good thing. This package.json is already very fat.